payments/views.py

```diff
@@ -0,0 +1,102 @@
+from django.core.mail import send_mail
+from django.shortcuts import redirect
+from django.views import View
+from django.http import HttpResponse
+from django.conf import settings
+from django.views.decorators.csrf import csrf_exempt
+from django.views.generic import TemplateView
+from users.models import Profile
+
+import stripe
+from rest_framework.utils import json
+
+stripe.api_key = settings.STRIPE_SECRET_KEY
+
+
+class ProductLandingPageView(TemplateView):
+    template_name = 'top_up.html'
+
+    def get_context_data(self, **kwargs):
+        context = super(ProductLandingPageView, self).get_context_data(**kwargs)
+        context.update({
+            "STRIPE_PUBLIC_KEY": settings.STRIPE_PUBLIC_KEY,
+
+            "product": 'Doładowanie punktów',
+        })
+        return context
+
+
+class CreateCheckoutSessionView(View):
+
+    def post(self, request, *args, **kwargs):
+        topup_value = request.POST.get('topup')
+        intent_value = int(topup_value) * 100
+        YOUR_DOMAIN = 'http://127.0.0.1:8000'
+        user = request.user
+
+        checkout_session = stripe.checkout.Session.create(
+            line_items=[
+                {
+                    'price_data': {
+                        'currency': 'usd',
```

Comment:
?

Reply:
Szymon (moze jakis tool do walut, w sumie nie wiemy moze wystarczy zahardcodowac pln - up to u)

Reply:
Narazie hardcoding. Jeśli projekt miałby wyjść poza PL to w formularzu wpisania kwoty dodamy listę walut doładowania.